packages/storage/src/implementation/list.ts

```diff
@@ -58,11 +59,15 @@ function fromBackendResponse(
     items: [],
     nextPageToken: resource['nextPageToken']
   };
+  const bucket = authWrapper.bucket();
+  if (bucket === null) {
```

Comment:
This is why I don't like mixing `==null` in the code. I have to look up `authWrapper.bucket()` signature to understand if it is actually meant to be `==` and made a mistake.